docs/source/usersguide/depletion.rst

```diff
@@ -370,3 +370,56 @@ The values of the one-group microscopic cross sections passed to
 :class:`openmc.deplete.IndependentOperator` are fixed for the entire depletion
 simulation. This implicit assumption may produce inaccurate results for certain
 scenarios.
+
+MSR-Continuous depletion
+===============================
+The :class:`~openmc.deplete.MsrContinuous` class adds the capability to add
+nuclide removal rates to a depletable material.
+
+An instance of this class requires
+:class:`~openmc.deplete.abc.TransportOperator` and :class:`~openmc.Model`
+instances, and can be passed directly to one of the Integrator classes::
+
+    ...
+    msr = openmc.deplete.msr.MsrContinuous(op, model)
+    integrator = openmc.deplete.PredictorIntegrator(op, time_steps, power,
+                msr_continuous=msr)
```

Comment:
When I implemented it in the first place I thought it would make sense to have everything in one place, but it could make sense now and shouldn't take long to re-adapt it this way if needed.